doc/api-extensions.md

```diff
@@ -1602,3 +1602,8 @@ Adds a new `GET /1.0/networks-acls/NAME/log` API to retrieve ACL firewall logs.
 
 ## storage\_zfs\_blocksize
 Introduces a new `zfs.blocksize` property for ZFS storage volumes which allows to set volume block size.
+
+## client\_join\_token
+This introduces client tokens which can be used for adding new remotes in a safer way.
```

Comment:
Should be `This adds token based certificate addition to the trust store as a safer alternative to a trust password`.

`remotes` are a concept specific to the Go client so probably best to avoid it be mentioned on the server side.